### DISCUSSION TITLE
areas/community

### DISCUSSION THREAD

**Discussion by giscus** [thepopupschool/thepopupschool#7] [Courses]
# tags/community

We Need to Watch Each Other Grow .

https://thepopupschool.org/tags/community

**Comment by kevindeland** [comment]
First!

**Reply by bonnittaroy** [reply]
VIDEO! VIDEO! VIDEO!

**Comment by namjul** [comment]
We had a first dialog https://www.youtube.com/watch?v=mvPiMSWM4Zk session.

**Reply by Phil18181** [reply]
Hello Pop-Up Legends, I really enjoyed this video and I am wondering what you are up to and whether I could participate. I still have a deep wish to go through the Pop-Up School material from scratch. I'm hesitant still because I am still struggling with my recovery and don't want to commit to something if it is not likely that I can follow through and stick with it. That's a capacity I do normally have though and it is easier to imagine it going well working with a group like this... Is that what you are doing, revisiting all of the material chronologically?

**Reply by namjul** [reply]
Hello Phil, we are in the process of planning a cohort with a structure to go over https://thepopupschool.org/course/new-theory-of-the-body/ chronologically. I will include you in the next letter.

**Reply by Phil18181** [reply]
Thank you!

**Comment by namjul** [comment]
We came together sharing our ideas and coordinating further steps https://www.youtube.com/watch?v=W24jy7ZzcFU

**Reply by namjul** [reply]
Our first class https://www.youtube.com/watch?v=nK5gqP5wQrc

**Comment by namjul** [comment]
Recording of our second call on ANTB primers https://www.youtube.com/watch?v=0A0hwKeMgPQ

Some notes and chat-history can be viewed here https://memex.samho.xyz/thepopupschool.2025.02.14

**Reply by namjul** [reply]
I will keep adding the recordings into this thread.

From 2025-02-20 https://www.youtube.com/watch?v=_F0NUsObZj4

**Reply by namjul** [reply]
The text below is a response from @bonnittaroy

> I really enjoy watching (or skimming through) these videos.
I thought I would address a question Raj asks around the 18 minute mark -- about arousal energy needing somewher to go...

> The basic formula is arousal energy seeks actions that satisfy.
In our modern world, when we get aroused ... what are the actions that can satisfy?
When there are no actions that can satisfy, we bump up the energy to the narrative mind, start creating stories, drama, etc...

> When in fact, what we should do is psuh that arousal energy through the perceptual organs. If you are getting yelled at, you know you can't take the action to punch the person in the face. So instead of bumping that energy up into a story, you open your perceptual organs. You actually look at the person and see them for the first time. YOu might look at the room, and see it for the first time -- that it is built like a prison, for example. You might look at the classroom and see it for the first time -- as a kind of prison ...

> Then you gain insight, and the actions you can take will follow

**Reply by namjul** [reply]
From 2025-02-28 https://youtu.be/i2oYWdH-Z9E

**Reply by namjul** [reply]
Last on ANTB affects module 2025-03-06 https://youtu.be/vpkbpbwg5qU

**Reply by namjul** [reply]
Response from @bonnittaroy

> HI everyone
Since I watched the last video, I thought I might clarify some of the notion of the "depressive" states. They can hold a lot of energy, but the word "depressive" has to do with the way the compress, or contract all of reality into a single point -- its as if the entire reality lived in this one particular focal point-- of rage, grief, or dread -- and nothing else about the situation can come through.

> So this naturally bleeds into the module on perception. If you are caught in one of these, and you open your perception -- how much of the world are you letting in. The story you are telling may be factually true -- its not just a story -- but how much of the reality of the situation does your perception let in? In the perception module it talks about egocentric and allocentric views. If you revisit your description of reality -- what view is it taking in, and what views doesn't it take in?

**Reply by namjul** [reply]
ANTB Perception 2025-03-20 https://www.youtube.com/watch?v=fNsxuuQrG4A

**Reply by namjul** [reply]
ANTB Memory 2025-03-28 https://youtu.be/Hkgxz-74Umw

**Reply by namjul** [reply]
ANTB Imagination & Synaesthesia 2025-04-11 https://youtu.be/yUk4LghnDn4

**Reply by namjul** [reply]
ANTB Imagination & Synaesthesia 2025-04-17 https://www.youtube.com/watch?v=PtgqgCBkNcw

**Reply by namjul** [reply]
ANTB Imagination & Synaesthesia 2025-04-25 https://youtu.be/4tlDGnT67Qk

**Reply by namjul** [reply]
ANTB Representational Thinking, Narrative, Meaning & the Justification System 2025-05-01
 https://www.youtube.com/watch?v=JdZnhioFKBQ

@bonnittaroy wrote to me

> "Listening to the recording now (after the Harvard conference)
You said something about a ground to stand on (a nice metaphor!)
and yes, there is a ground to stand on, it's called the earth. Then we use that as a metaphor, in the sense "does the mind have a similar, metaphoric ground to stand on?" And of course, you see, there can't be any such ground, because the mind itself is ephemeral. But then, again, speaking metaphorically, the mind does have a ground to stand on (or more precisely grounds to stand on) but obviously, also, those grounds must be ephemeral too. A justification system is an ephemeral ground to stand on. The problem comes when we forget that it's ephemeral. And the "shock" that all justification systems are living, moving, ephemeral grounds (notice the phrase, he has grounds to complain)... means that you have subtle thought of ephemeral grounds as concrete, kinda forgetting that its a metaphor, that has slipped into reification (misplaced concreteness)

further

> This was followed by Roberta talking about action and ontology. Action is the bridge between the ephemeral (virtual perception, thought, abstraction) and the real (ontological). So when you get into the weeds about the ephemeral (metaphorical ) nature of representational thinking, instead of looking toward more abstraction to evaluate what you say (speech acts) and get stuck in endless justification systems, look instead toward the actions in the world that proceed from those speech acts. Here's a metaphor: use speech like a joystick in a video game. Eventually you get skillful at using it fluidly to direct action.

**Reply by namjul** [reply]
ANTB Representational Thinking, Narrative, Meaning & the Justification System 2025-05-09 https://youtu.be/wYqfsF0QedY

**Reply by namjul** [reply]
ANTB Math & Pure Abstractions 2025-04-25 https://youtu.be/N3R7XWM7was

**Reply by namjul** [reply]
ANTB Math & Pure Abstractions 2025-05-29 https://youtu.be/oj-XmU8x_gw

**Reply by namjul** [reply]
ANTB Awareness: Reflection, Recursivity, Reflexivity 2025-06-12 https://youtu.be/7XSpFRL3Kuc

**Reply by namjul** [reply]
ANTB Awareness: Reflection, Recursivity, Reflexivity 2025-06-27 https://youtu.be/QApduO3o4EE
